Sources/Mugen/MugenNotificationCenter.swift

```diff
@@ -63,7 +63,7 @@ public struct MugenNotificationCenter {
     }
 }
 
-public enum DataSource {
+public enum DataSource: @unchecked Sendable {
     case userDefaults(UserDefaults)
     case fileStorage(
         url: URL = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask)[0]
```

Comment:
This is not thread-safe, so the `@unchecked Sendable` should not be applied to it.